pymatgen/io/openmm/utils.py

```diff
@@ -341,130 +365,247 @@ def assign_charges_to_mols(
 
 
 def parameterize_system(
-    topology: Topology,
-    smile_strings: List[str],
-    box: List[float],
-    force_field: Union[str, Dict[str, str]] = "sage",
-    partial_charge_method: str = "am1bcc",
-    partial_charge_scaling: Dict[str, float] = None,
-    partial_charges: List[Tuple[Union[pymatgen.core.Molecule, str, Path], np.ndarray]] = [],
+        topology: Topology,
+        smile_strings: List[str],
+        box: List[float],
+        force_field: Union[str, Dict[str, str]] = "sage",
+        partial_charge_method: str = "am1bcc",
+        partial_charge_scaling: Dict[str, float] = None,
+        partial_charges: List[
+            Tuple[Union[pymatgen.core.Molecule, str, Path], np.ndarray]] = [],
 ) -> openmm.System:
     """
     Parameterize an OpenMM system.
 
     Args:
         topology: an OpenMM topology.
         smile_strings: a list of SMILEs representing each molecule in the system.
-        box: list of [xlo, ylo, zlo, xhi, yhi, zhi].
-        force_field: name of the force field. Currently only Sage is supported.
-        partial_charge_method:
-        partial_charge_scaling:
-        partial_charges:
+        box: list of [xlo, ylo, zlo, xhi, yhi, zhi] in nanometers.
+        force_field: Name of the force field or dict of forcefields for each
+                    small molecule, e.g. {"O": "spce"}. Small molecule
+                    forcefields and water models can either be provided
+                    informally, i.e. "gaff" or "sage" for small molecules,
+                    or "spce", "tip3p", or "tip4p" for water, or can be
+                    formally defined with OpenMM filenames,
+                    e.g. "charmm36/water.xml". Large molecule forcefields
+                    must be specified with the full path,
+                    e.g. "amber14/protein.ff14SB.xml".
+        partial_charge_method: Method for OpenFF partial charge assignment
+                                for small molecules without charges provided
+                                in partial_charges
+        partial_charge_scaling: Scaling for partial charges, either
+                                uniformly as a float or per molecule as a dict
+                                of {str: float, . . .}, e.g. {"[Li+]": 0.8}
+        partial_charges: List of tuples of (molecule, charges).
+                        The Molecule can be a Pymatgen Molecule or the
+                        path to a structure file that can be parsed by
+                        Pymatgen. Charges is a numpy array with length equal
+                         to the number of sites in the molecule.
 
     Returns:
         an OpenMM.system
     """
 
     partial_charge_scaling = partial_charge_scaling if partial_charge_scaling else {}
     partial_charges = partial_charges if partial_charges else []
-    supported_force_fields = ["Sage"]
+    basic_water_ffs = ["tip3p", "spce", "tip4p"]
+    basic_small_ffs = ["gaff", "sage"]
+
+    all_small_ffs = [
```

Comment:
Could we get rid of this list and make list generation programmatic?

Reply:
Should be good now